### PR TITLE
feat(checkbox-group): support label-key and value-key prop and export checked for default slot

### PR DESCRIPTION
**Pull request by chouchouji**
## Checklist

List of tasks you have already done and plan to do.

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

Describe your modifications here.

## Issues

The issues you want to close, formatted as close #1.

## Related Links

Links related to this pr.

### CODE REVIEW

File: packages/varlet-ui/src/checkbox-group/props.ts [hunk 1]
```diff
@@ -18,6 +19,8 @@ export const props = {
   },
   max: [String, Number],
   options: Array as PropType<Array<CheckboxGroupOption>>,
+  labelKey: String,
+  valueKey: String,
```
Comment (haoziqaq) [inline packages/varlet-ui/src/checkbox-group/props.ts:23]: 这里直接给默认值吧。

File: packages/varlet-ui/src/checkbox-group/CheckboxGroup.vue [hunk 2]
```diff
@@ -35,6 +37,8 @@ export default defineComponent({
     const max = computed(() => props.max)
     const checkedCount = computed(() => props.modelValue.length)
     const checkboxGroupOptions = computed(() => (isArray(props.options) ? props.options : []))
+    const valueField = computed(() => props.valueKey ?? 'value')
+    const labelField = computed(() => props.labelKey ?? 'label')
```
Comment (haoziqaq) [inline packages/varlet-ui/src/checkbox-group/CheckboxGroup.vue:41]: 给了默认值这边应该就不需要了。

File: packages/varlet-ui/types/checkbox.d.ts [hunk 3]
```diff
@@ -30,7 +30,7 @@ export class Checkbox extends VarComponent {
   $props: CheckboxProps
 
   $slots: {
-    default(): VNode[]
+    default(checked: boolean): VNode[]
```
Comment (haoziqaq) [inline packages/varlet-ui/types/checkbox.d.ts:33]: 这里类型有点问题，参考 countdown

Comment (chouchouji) [reply packages/varlet-ui/types/checkbox.d.ts:33]: 哦哦哦，应该是对象

Comment (chouchouji) [reply packages/varlet-ui/types/checkbox.d.ts:33]: 这个需要单独拉出来一个type吗？

**Comment by chouchouji** [comment]
耗子哥，支持label-key和value-key的话，类型这里label和value也不是必填的了
image

**Review by haoziqaq — APPROVED** [review]

**Pull request merged**